client-v2/src/components/Feed.js

```diff
@@ -99,31 +117,33 @@ class Feed extends React.Component<FeedProps, FeedState> {
                     <LoaderDisplay loading={!value && pending}>
                       <div>
                         {value &&
-                          value.response.results.map(
-                            ({
-                              webTitle,
-                              webUrl,
-                              webPublicationDate,
-                              elements,
-                              fields,
-                              frontsMeta: { tone }
-                            }) => (
-                              <FeedItem
-                                key={webUrl}
-                                title={webTitle}
-                                href={webUrl}
-                                publicationDate={webPublicationDate}
-                                tone={tone}
-                                trailText={fields && fields.trailText}
-                                thumbnailUrl={
-                                  elements && getThumbnail(elements)
-                                }
-                                internalPageCode={
-                                  fields && getId(fields.internalPageCode)
-                                }
-                              />
-                            )
-                          )}
+                          value.response.results
+                            .filter(result => result.webTitle)
```

Comment:
Despite the diff, the only change here is this line right?

Reply:
Yep